src/app/find-help/reply-offer/reply-offer.component.html

```diff
@@ -63,10 +63,16 @@ <h3>{{ "REPLY_OFFER" | translate }}</h3>
           (change)="onConsentChange()"
           name="consent"
         >
-          <span>{{ "ACCEPT" | translate }} </span
-          ><a routerLink=""
+          <span>{{ "ACCEPT" | translate }} </span>
+          <a target="_blank" href=""
             ><span>{{ "CONSENT" | translate }}</span></a
           >
+          <span *ngIf="isLoggedIn === false">
```

Comment:
To jest `boolean`, więc nie trzeba wprost porównywać do `false`, wystarczy:
```suggestion
          <span *ngIf="isLoggedIn">
```

Reply:
Ogólnie preferuje nie tworzyć dodatkowych metod w komponencie, ktore robią to samo co serwis.
Użyj po prostu serwisu authService.isLoggedIn w htmlu

Reply:
w tym wypadku raczej !isLoggedIn ale serio to problem?

Reply:
To był nieblokujący komentarz.

Reply:
@bbartosiak poprawione